src/NftDetails/NftDetails.tsx

```diff
@@ -123,13 +125,13 @@ const NftOverview = ({nft}: {nft: YoroiNft}) => {
       <HR />
 
       <MetadataRow title={strings.description}>
-        <Text secondary>{nft.description}</Text>
+        <Text secondary>{normalizeMetadataString(nft.description)}</Text>
       </MetadataRow>
 
       <HR />
 
       <MetadataRow title={strings.author}>
-        <Text secondary>{nft.metadata.originalMetadata.author ?? '-'}</Text>
+        <Text secondary>{normalizeMetadataString(nft.metadata.originalMetadata?.author)}</Text>
```

Comment:
In this PR https://github.com/Emurgo/yoroi-mobile/pull/2396, the optional chain operator. Should be removed here too? or added there? for consistency